lms/djangoapps/verify_student/models.py

```diff
@@ -1237,6 +1237,11 @@ class VerificationAttempt(TimeStampedModel):
         blank=True,
     )
 
+    @property
+    def status_changed(self):
```

Comment:
I believe that `StatusModel` is what puts a `status_changed` field on the model. Could you explain the need for this property? With this property, theoretically, we don't need to inherit from `StatusModel`. That said, this property doesn't work the same was as the `status_changed` field on other IDV models. On this model, it will always return the `created` field, which will never changed, whereas on other models, it'll return the date the `status_field` changed, which will change. For that reason, it may make sense to still inherit from `StatusModel` and use the default implementation of `status_changed` instead of overriding it with this property.

Note that if you do inherit from `StatusModel`, you mya have to change how the status is defined as well (i.e. `status` should be `STATUS`; see the [docs](https://django-model-utils.readthedocs.io/en/latest/models.html#statusmodel).